website/components/blog/Categories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react"
+import Link from "next/link"
+
+export interface Props {
+  tags: Array<string>
+  currentTag?: string
+}
+
+const Categories: React.FC<Props> = ({ tags, currentTag = "All" }) => {
+  return (
+    <aside>
+      <h2>Categories</h2>
+      <ul className="list-unstyled mt-4">
+        <li>
+          <Link href={`/blog`}>
+            <a className={`${currentTag === "All" ? "text-gradient" : ""}`}>All</a>
```

Comment:
I think we have classnames library for conditional styling, but this is fine as well.